typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumInevmUSDCWarpConfig.ts

```diff
@@ -0,0 +1,45 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+  buildAggregationIsmConfigs,
+  defaultMultisigConfigs,
+} from '@hyperlane-xyz/sdk';
+
+import { tokens } from '../../../../../src/config/warp.js';
+
+export const getEthereumInevmUSDCWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  // TODO: seems to be evidence that this ISM should have been set https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3233/commits/dc8d50c9c49cdea8417fbe9dad090dc13f078fff
```

Comment:
The commit referenced here suggests that the intention was to deploy the warp route with an configured ISM, the checker tooling suggests that it was not

Reply:
Forgot to commit a fix I had for this locally, my bad